generators/server/templates/build.gradle.ejs

```diff
@@ -264,10 +264,10 @@ repositories {
     mavenLocal()
     mavenCentral()
     jcenter()
-    <%_ if (!SPRING_BOOT_VERSION.endsWith('RELEASE')) { _%>
+    <%_ if (!SPRING_BOOT_VERSION.endsWith('RELEASE') || SPRING_BOOT_VERSION.startsWith('2.4')) { _%>
```

Comment:
Is it always mandatory?

Reply:
Spring Boot changed their versioning in 2.4 so `.RELEASE` is not part of GA releases anymore. I haven't seen any logs that indicate outdated properties in my tests so this can probably be removed. Maybe we should change this so it's only added for Milestone releases?

Reply:
The test should should be changed to match milestone (e.g .M1) or RC (e.g. .RC1) releases:
```suggestion
    <%_ if (!!SPRING_BOOT_VERSION.match(/\.(M|RC)\d$/)) { _%>
```